components/KickoffRequestView/KickoffRequestPanel.tsx

```diff
@@ -0,0 +1,71 @@
+import { useState } from 'react';
+import { useRecoilValue } from 'recoil';
+import { selectedMeasureState } from '../../atoms/selectedMeasure';
+import { Card, Tabs, Title, Space } from '@mantine/core';
+import KickoffHeaders from './KickoffHeaders';
+import KickoffBody from './KickoffBody';
+import { exportUrlState } from '../../atoms/exportUrl';
+import KickoffPostUrl from './KickoffPostUrl';
+
+export default function KickoffRequestPanel() {
+  const selectedMeasure = useRecoilValue(selectedMeasureState);
+  const exportUrl = useRecoilValue(exportUrlState);
+  const [activeTab, setActiveTab] = useState(0);
+  let body: fhir4.Parameters | undefined;
+  let headers: string | undefined;
+
+  if (selectedMeasure && exportUrl) {
+    [body, headers] = buildKickoffRequest(selectedMeasure, exportUrl);
+  }
```

Comment:
I think it makes more sense to use this if statement inside a useEffect and store the body and headers variables as pieces of state, and only run the effect on selectedMeasure or exportUrl change. Otherwise this code will rerun every time a tab is changed and force additional re-rendering.

Reply:
Whoops, didn't see this. [My comment](https://github.com/projecttacoma/bulk-submit-demo/pull/6#discussion_r867020206) is addressing this same issue in a slightly different way, lmk what you think

Reply:
Switched to use a selector in bb84cdb.